src/open_inwoner/accounts/templates/accounts/registration_necessary.html

```diff
@@ -1,13 +1,15 @@
 {% extends 'master.html' %}
-{% load i18n static form_tags card_tags grid_tags %}
+{% load i18n static form_tags card_tags grid_tags solo_tags %}
 
 {% block content %}
 
 <div class="registration-grid">
 {% render_grid %}
     {% render_column span=9 %}
         {% render_card tinted=True  %}
+            {% get_solo 'configurations.SiteConfiguration' as config %}
             <h1 class="h1">{% trans "Registratie voltooien" %}</h1><br>
+            {% if config.registration_text %}<p class="p">{{ config.registration_text|urlize|linebreaksbr }}</p><br>{% endif %}
```

Comment:
Perhaps add an `id` or something specific to the `<p>` element so you can target it directly in the test. See comment below.

Reply:
Ideally we'd have locators and their assertions like in Playwright so we can target by role or CSS rules.